pom.xml

```diff
@@ -345,7 +311,89 @@
+                            <systemPropertyVariables>
+                                <rabbitmq.host>localhost</rabbitmq.host>
+                                <rabbitmq.port>5672</rabbitmq.port>
+                                <rabbitmq.username>myuser</rabbitmq.username>
+                                <rabbitmq.password>myuser</rabbitmq.password>
+                                <rabbitmq.exchange>ei-exchange</rabbitmq.exchange>
+                                <rabbitmq.key>#</rabbitmq.key>
+                            </systemPropertyVariables>
+                            <excludes>
+                                <exclude>**/systemtest/**</exclude>
+                            </excludes>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>integration-test</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <phase>integration-test</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <!-- The Configuration of the system-test profile -->
+        <profile>
+            <id>systemTest</id>
+            <properties>
+                <skipUTs>true</skipUTs>
+            </properties>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>2.22.0</version>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <ei.config.properties.file.path>${systemtestsProperties}</ei.config.properties.file.path>
+                            </systemPropertyVariables>
+                            <excludes>
+                                <exclude>**/frontend/*</exclude>
+                            </excludes>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>systemTest</id>
+                                <phase>integration-test</phase>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```

Comment:
This profile is duplicated, remove one of them :::
<!-- The Configuration of the system-test profile -->
        <profile>
            <id>systemTest</id>
            <properties>
                <skipUTs>true</skipUTs>
            </properties>
            <activation>
                <activeByDefault>false</activeByDefault>
            </activation>
            <build>
                <plugins>
                    <plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-failsafe-plugin</artifactId>
                        <version>2.22.0</version>
                        <configuration>
                            <systemPropertyVariables>
                                <ei.config.properties.file.path>${systemtestsProperties}</ei.config.properties.file.path>
                            </systemPropertyVariables>
                            <excludes>
                                <exclude>**/frontend/*</exclude>
                            </excludes>
                        </configuration>
                        <executions>
                            <execution>
                                <id>systemTest</id>
                                <phase>integration-test</phase>
                                <goals>
                                    <goal>integration-test</goal>
                                    <goal>verify</goal>
                                </goals>
                            </execution>
                        </executions>
                    </plugin>
                </plugins>
            </build>
        </profile>